packages/core/src/payment/payment-strategy-registry.spec.ts

```diff
@@ -105,10 +101,6 @@ describe('PaymentStrategyRegistry', () => {
             );
         });
 
-        it('returns offsite strategy if none is registered with method name and method is hosted', () => {
-            expect(registry.getByMethod(getAdyenAmex())).toBeInstanceOf(OffsitePaymentStrategy);
-        });
-
```

Comment:
I think it still should be tested or we may add an offsite e2e test.
The rest are all good.

Reply:
This was pretty much a dummy test from what I understood. But you are right we should write an offsite test and we do have a task to do so as well.